packages/mobile/src/harmony-native/components/TextLink/TextLink.tsx

```diff
@@ -73,28 +74,60 @@ export const TextLink = <ParamList extends ReactNavigation.RootParamList>(
     )
   }))
 
+  let element = (
+    <AnimatedText
+      style={[
+        animatedLinkStyles,
+        css({
+          textDecorationLine: isPressing || showUnderline ? 'underline' : 'none'
+        })
+      ]}
+      variant={textVariant}
+      {...other}
+    >
+      {children}
+    </AnimatedText>
+  )
+
+  const rootProps = {
+    onPressIn: () => setIsPressing(true),
+    onPressOut: () => setIsPressing(false)
+  }
+
+  if (to) {
+    element = (
+      <Link to={to} action={action} onPress={handlePress} {...rootProps}>
```

Comment:
we should spread the props right?
```suggestion
      <Link to={to} action={action} onPress={handlePress} {...rootProps} {...other}>
```

Reply:
yes def, i have a lot more text-link improvements coming so i will ignore this for this pr